ui/src/App.tsx

```diff
@@ -1,25 +1,17 @@
-import React, { useReducer } from 'react';
-import { AppDispatchContext, AppDispatchReducer, initialState } from './reducers/AppDispatchReducer';
-import './App.css';
-import MemeUpload from './MemeUpload';
-import ImageUpload from './ImageUpload';
+import React from 'react';
+import {Route, Switch, BrowserRouter } from "react-router-dom";
```

Comment:
Convention is to import `BrowserRouter as Router`, then just render `Router`.

Reply:
ok sad face. I will fix it now